### PR TITLE
Added Log4J for internal CT instance

### PR DESCRIPTION
**Pull request by abhimanyugupta07**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.2.2] - TBD
```
Comment (massdosage) [inline CHANGELOG.md]: Similar to other PR, let's go with 1.3.0

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
...
+### Added
+- Log4j Xml for internal Circus Train instance.
```
Comment (massdosage) [inline CHANGELOG.md]:
```suggestion
- Log4j XML for internal Circus Train instance.
```

File: README.md [hunk 3]
```diff
@@ -13,6 +13,7 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 | aws\_region | AWS region to use for resources. | string | n/a | yes |
 | cpu | The number of CPU units to reserve for the Shunting Yard container. Valid values can be 256, 512, 1024, 2048 and 4096. Reference: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html | string | `"1024"` | no |
 | ct\_common\_config\_yaml | Common Circus Train configuration to be passed to internal Circus Train instance. It can be used, for example to configure Graphite for Circus Train. Refer to [Circus Train README](https://github.com/HotelsDotCom/circus-train/blob/master/README.md) for an exhaustive list of options supported by Circus Train. | string | n/a | yes |
+| ct\_log4j\_xml | Log4j Xml file to be passed to internal Circus Train instance to control log level. | string | n/a | yes |
```
Comment (massdosage) [inline README.md]:
```suggestion
| ct\_log4j\_xml | Log4j XML file to be passed to internal Circus Train instance to configure logging. | string | n/a | yes |
```

File: variables.tf [hunk 4]
```diff
@@ -67,6 +67,11 @@ variable "ct_common_config_yaml" {
   type        = "string"
 }
 
+variable "ct_log4j_xml" {
+  description = "Log4j Xml file to be passed to internal Circus Train instance to control log level."
```
Comment (massdosage) [inline variables.tf]:
```suggestion
  description = "Log4j XML file to be passed to internal Circus Train instance to configure logging."
```

**Review by massdosage — CHANGES_REQUESTED** [review]

**Review by max-jacobs — APPROVED** [review]

**Review by massdosage — APPROVED** [review]

**Pull request merged**